scripts/test/Muon/results_tab_widget/results_tab_presenter_test.py

```diff
@@ -140,6 +141,21 @@ def test_results_table_request_with_empty_results_does_nothing(self):
 
         self.assertEqual(0, self.mock_model.create_results_table.call_count)
 
+    def test_that_disable_observer_calls_on_view_when_triggered(self):
+        presenter = ResultsTabPresenter(self.mock_view, self.mock_model)
+        disable_notifier = GenericObservable()
```

Comment:
see above

Reply:
The view in the setup has already been mocked out unlike in phase table so does not have right methods.